.github/workflows/go.yml

```diff
@@ -0,0 +1,24 @@
+name: Go
```

Comment:
this will run the Go unit tests on updates to the master branch, and on PRs (but it won't block merges if they are failing or anything)

Reply:
Excellent.